jobs/issues_status.rb

```diff
@@ -76,4 +108,22 @@
              trend_class: trend_class_closed,
              arrow: 'icon-arrow-' + trend_class_closed
             )
+
+  # ----------- PULL REQUEST STATS -------------------------- #
+
+  # rubocop:disable Style/RescueModifier
```

Comment:
interesting - disabling then re-enabling rubocopy for two lines of code - why not just write the code in a format that rubocop likes?